examples/components/styles.js

```diff
@@ -1,6 +1,7 @@
 import '../../src/more-or-less/moreOrLess.scss';
 import '../../src/flex/flexView.scss';
 import '../../src/loading-spinner/style.scss';
+import '../../src/dropdown/dropdown.scss';
 import '../../src/confirmation-input/confirmationInput.scss';
 import '../../src/badge/badge.scss';
 import '../../src/toggle/toggle.scss';
```

Comment:
Completely didn't seen. Thank you for that.

PS: you want me to add here the css mentioned in #108 on in another pr?